test/check-bootstrap.bash

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+
+function line() {
```

Comment:
What about:

- pull line() into common.bash
- doing what diagnose.bash does
- pulling execute() out of diagnose.bash into common.bash
- put BLUE there

?
- use

Reply:
I'm not sure what you mean by "use" in the last point

Reply:
What're your thoughts on the latest version?

Reply:
Sorry, I didn't finish the comment. I meant to write "use the execute function for echoing a command before running it.

Other than that, this looks good.

Reply:
If we use the `execute` function, stdout will get polluted with the names of all the tests and the test results (for example if we ran `execute/*.v3). Do you want to have that? The current version pipes it all out to `/dev/null` to reduce the clutter.

Reply:
Ok, I guess it's alright.